pkg/cmd/get_clustercompliancereport.go

```diff
@@ -49,8 +51,15 @@ func NewGetClusterComplianceReportsCmd(executable string, cf *genericclioptions.
 			if err != nil {
 				return err
 			}
-			// generate compliance and compliance failure detail reports
-			complianceMgr := compliance.NewMgr(kubeClient, logger)
+			kubeClientset, err := kubernetes.NewForConfig(kubeConfig)
+			if err != nil {
+				return err
+			}
+			starboardConfig, err := starboard.NewConfigManager(kubeClientset, starboard.NamespaceName).Read(ctx)
+			if err != nil {
+				return err
+			}
+			complianceMgr := compliance.NewMgr(kubeClient, logger, starboardConfig)
```

Comment:
Accepting it for now, but as I mentioned in my first review, when you inject dependencies limit them to absolute minimum. Otherwise you are introducing unnecessary coupling between two Go packages, i.e. starboard and compliance. The starboard configuration object configured other things, not just compliance.